### PR TITLE
Link to blog from home

### PR DESCRIPTION
**Pull request by ajyong**
## Tablet / Landscape Mobile
image

## Mobile
image

## Desktop
image

It looks a little busy in desktop, but we can always refine this later.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,13 +53,16 @@ <h1>CODEVID-19</h1>
       </p>
       <ul class="actions">
         <li>
-          <a href="join-the-fight" class="button">Join the Fight</a>
+          <a href="{{ '/join-the-fight' | relative_url }}" class="button primary">Join the Fight</a>
```
Comment (MarkBennett) [inline index.html]: Let's not change this link title in this PR as I think that will open a can of worms and keep this from going out.

File: index.html [hunk 2]
```diff
@@ -53,13 +53,16 @@ <h1>CODEVID-19</h1>
...
         </li>
         <li>
           <a href="#become-a-sponsor" class="button scrolly"
             >Become a Sponsor</a
           >
         </li>
+        <li>
+          <a href="{{ '/blog' | relative_url }}" class="button">Read Our Blog</a>
```
Comment (MarkBennett) [inline index.html]: 👍

**Review by MarkBennett — APPROVED** [review]
Approved as is. Ship it @ajyong!

File: index.html (hunk 1 above)
Comment (ajyong) [reply index.html]: Before I do anything, I wanted to make sure my understanding is correct here.  I kept the title the same: did you mean the button style?  I can revert that no problem.

Where I'm becoming confused is when to use `relative_url`.  To me, I understand its benefits (if you use it in a `_layout/` file, it works for all pages), but maybe I should revert the use of it in this specific context.  I don't think we're going to be moving the landing page into a folder during the lifespan of this hackathon.  Overall I feel I was adding this to be "safe" from an unlikely scenario.

File: index.html (hunk 2 above)
Comment (ajyong) [reply index.html]: Same question I have about the necessity of `relative_url`.  tl;dr: it looks useful in `_layouts/`-level HTML, but not here.

Comment (MarkBennett) [reply index.html]: The only other time it's useful is when you're serving it from a not root url. I often use it as a rule, but you're right that `relative_url` is not needed if a standard relative link (i.e. `blog.html` or `./blog.html`) would work.

File: index.html (hunk 1 above)
Comment (MarkBennett) [reply index.html]: Hey, to address the first comment I'm specifically talking about the link title. I realize it wasn't changed but changing it was mentioned so I wanted to way in.

File: index.html (hunk 2 above)
Comment (ajyong) [reply index.html]: I'll leave them in until I've become a bit more familiar with Jekyll templates.

File: index.html (hunk 1 above)
Comment (ajyong) [reply index.html]: Oh okay.  I'll leave this as-is then.

**Comment by ajyong** [comment]
If we merge this with #79, having 4 buttons on the landing section feels too visually busy.  It's not the end of the world but I would like us to pick 3.

**Comment by ajyong** [comment]
Oof, okay, this will definitely need a new review and a tweak.  The template was not designed to support more than 3 buttons horizontally:
image

This happens on tablet (portrait) and mobile (landscape).  Even adding `actions fit` to the tag's class did not fix it because our button labels are too long.  Applying a `flex-wrap: wrap` on the container also looks awful:
image

Can we make the tough decision of removing a button?  My vote would be on sponsors, as painful as that sounds.  There is a call-for-action in Join the Fight for more sponsors, and we also have a section devoted to them in the landing page.

**Comment by MandyMeindersma** [comment]
weird, If you are using list item like you are there would be a space between the two buttons vertically, cry

I am in favour of removing sponsors

**Comment by ajyong** [comment]
Actually, I think I got a SASS fix for this.  Testing it out now and going to make a separate PR once I verify it doesn't cause regressions:
image

Basically if something flex-wraps to a second row, it will take up the entire width of that row, otherwise it does what happens to the row above and shares the width:
image
  
It's not ideal but at least the gutters between the buttons look better, and again, this is in a pinch.

**Review by MandyMeindersma — APPROVED** [review]
beautiful

**Pull request merged**